docs/source/new-project.rst

```diff
@@ -583,31 +581,29 @@ A callback is an arbitrary self-contained program that can be executed at arbitr
 
 Here's an example adding a not-so-fancy learning rate decay rule:
 
-.. code-block:: python
+.. testcode::
 
-    class DecayLearningRate(pl.Callback)
+    class DecayLearningRate(pl.callbacks.Callback):
```

Comment:
the previous was correct and should work. Callback is getting imported in pytorch_lightning/init.py